extension/assets/css/HFX.css

```diff
@@ -6,3 +6,486 @@
   margin-bottom: 15px;
```

Comment:
Isn't the extension folder the equivalent of the build/ release folder?
And shouldn't be in git?

Reply:
No. Some of the folders *within* the extension folder are build/release folders and aren’t included. There is probably better structuring/management of what is in the extensions folder to be done in the future.